app-frontend/src/app/pages/settings/settings.html

```diff
@@ -0,0 +1,15 @@
+<div class="container column-stretch dashboard">
+  <div class="sidebar">
+    <div class="sidebar-scrollable">
+      <nav>
+        <a ui-sref="settings.profile" ui-sref-active="active">Profile</a>
+        <a ui-sref="settings.account" ui-sref-active="active">Account</a>
+        <a ui-sref="settings.tokens" ui-sref-active="active">API tokens</a>
```

Comment:
There are a few sref links such as this that don't have routes defined. Not sure if that's intentional at this stage, but wanted to bring it up.